app/gcl/associate-cloud-engineer/domain4/domain4.css

```
@@ -0,0 +1,714 @@
+    align-items: center;
+    gap: 12px;
+    font-size: 20px;
+    font-weight: 800;
+    color: var(--d4-text-bright, #ebd8ec);
+    letter-spacing: -0.01em;
+    margin-bottom: 28px;
+    padding-bottom: 20px;
+    border-bottom: 1px solid var(--d4-border, #2b1a30);
+}
+
+/* h2.stitle 内の ::before を無効化（通常 stitle は小見出し用）*/
```

Stylelint (17.6.0)

[error] Expected whitespace before "*/" (comment-whitespace-inside)

(comment-whitespace-inside)